data/config/color.py

```diff
@@ -10,8 +10,6 @@ def color(self, keypath):
 
     def graphics_color(self, keypath):
         color = self.color(keypath)
-        if not color:
-            color = self.color("default.text")
```

Comment:
Does this mean that the default.text config is useless now?

Reply:
No, there's still one case where we might have an unknown team, I think we see it during spring training

https://github.com/MLB-LED-Scoreboard/mlb-led-scoreboard/blob/1ac208b7668e101305730a6565550182bfccc32c/data/scoreboard/team.py#L15-L32

Reply:
I meant the [one for the scoreboard colors](https://github.com/MLB-LED-Scoreboard/mlb-led-scoreboard/blob/master/colors/scoreboard.json.example#L300-L303), not the team

Reply:
Oh, yes. I added text nodes to everything that didn't have one so that it was explicit